### ISSUE TITLE
how to upgrade boostrap version to 4.0 beta

### ISSUE THREAD

**Issue by zoujiaqing** [ComBoot/ComBoot#37]
like title :)

**Comment by TheZoker** [comment]
Upgrading to bootstrap 4 is not trivial. It requires a lot of work, because they changed a lot of components. I'll be working on an upgrade soon, but can't give an ETA when it will be finised.

**Comment by zoujiaqing** [comment]
@TheZoker 👍 
bootstrap v4 is released! :)

**Comment by TheZoker** [comment]
Yes I'm aware of that and currently working on a 4.0.0 compatible version :)

**Comment by cime** [comment]
@TheZoker do you have any idea when it would be ready to use (or test)?

**Comment by zoujiaqing** [comment]
Hi @TheZoker , bootstrap 4.1.x is released :)
